test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/FlushReportingStream.cs

```diff
@@ -11,11 +13,20 @@ public static class FlushReportingStream
     {
         public static Stream GetThrowingStream()
```

Comment:
This method isn't called in this PR. Why is it changing here?

Reply:
I wanted to write the test where I take in a query string to conditionally indent data(the tests in this PR), but since mock objects were being used I wasn't able to do that (since the mock setup only created setup 'Get' part). Later found that this piece of code here never worked as it was supposed as the assignment of this stream was just no-op (example: https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/XmlSerializerOutputFormatterTest.cs#L358)
So I updated the code to use DefaultHttpContext and changed this stream to be simple.